### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by iamvasuh**
replaced share with skunkworks-crow

### CODE REVIEW

**Comment by iamvasuh** [comment]
share is replaced with skunkworks-crow

**Comment by iamvasuh** [comment]
ok done

**Comment by iamvasuh** [comment]
done now @shobhitagarwal1612

File: README.md [hunk 1]
```diff
@@ -114,21 +114,21 @@ Is user wants to manually connect to it then the screen shows the network name s
...
 
 A receiver will need to connect to the network the sender has created. There are two ways to connect to the network:
 A QR code generated by the sender can be scanned by a receiver, it will get all the information required to connect to the sender (wifi name, port,  and password if the connection is protected).
 Tap on the wifi network available to connect and after successful connection enter port number to start receiving the forms.
 
 Connecting to the sender’s network will disable all the wifi network available so that no other network can interrupt the current connection. After receiving the forms it will restore the last saved state.
 
-For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-SHARE-odk-share” is set as the default network name and for versions >= 8 it is AndroidShare-XXXX which is set by Android.
+For some devices, the user is not allowed to configure the wifi hotspot from the app. Currently for Android versions < 8, “ODK-skunkworks-crow-odk-skunkworks-crow” is set as the default network name and for versions >= 8 it is Androidskunkworks-crow-XXXX which is set by Android.
```
Comment (lakshyagupta21) [inline README.md]: here ODK-SHARE-odk-share as well as AndroidShare-XXXX should not be changed because nothing is changed in code so it will still start the hotspot with the same name. What do you say  about this? @shobhitagarwal1612

**Comment by lakshyagupta21** [comment]
@iamvasuh In case template is not coming in your PR now, then use this https://github.com/opendatakit/skunkworks-crow/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: README.md (hunk 1 above)
Comment (shobhitagarwal1612) [reply README.md]: Agreed @lakshyagupta21 

This change should be reverted from this PR and handled separately in a new PR. @lakshyagupta21 can you open a new issue?

Comment (lakshyagupta21) [reply README.md]: Sure

**Comment by lakshyagupta21** [comment]
@iamvasuh Lets not do all the changes in the same branch. 
There are seperate issue filed for the latest commit https://github.com/opendatakit/skunkworks-crow/issues/136 and https://github.com/opendatakit/skunkworks-crow/issues/140. We'll do the changes in the different PR. Lets push only readme file changes in this PR.

**Comment by lakshyagupta21** [comment]
And #136 is already claimed by someone else so please don't pick issues which are assigned to someone else.

**Comment by iamvasuh** [comment]
> @iamvasuh Lets not do all the changes in the same branch.
> There are seperate issue filed for the latest commit #136 and #140. We'll do the changes in the different PR. Lets push only readme file changes in this PR.

ok then

**Comment by iamvasuh** [comment]
> And #136 is already claimed by someone else so please don't pick issues which are assigned to someone else.

yup i will make sure about that

File: README.md (hunk 1 above)
Comment (shobhitagarwal1612) [reply README.md]: Please Undo this change

File: share_app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Share</string>
```
Comment (shobhitagarwal1612) [inline share_app/src/main/res/values/strings.xml]: Please undo this change

File: share_app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -86,7 +86,7 @@
     <string name="add_feedback">Add Feeback</string>
     <string name="view_form_in_collect">View in ODK Collect</string>
     <string name="collect_not_installed">Install ODK Collect to view the form</string>
-    <string name="install_collect">Share app requires ODK Collect to be installed in the device</string>
+    <string name="install_collect">skunkworks-crow app requires ODK Collect to be installed in the device</string>
```
Comment (shobhitagarwal1612) [inline share_app/src/main/res/values/strings.xml]: Please undo this change

**Comment by iamvasuh** [comment]
@shobhitagarwal1612  can u merge this PR  and then merge #150  PR . ?

**Comment by shobhitagarwal1612** [comment]
Hey. You can update the existing PR by pushing to the same branch instead of opening a new PR with changes
